src/Georadix.WebApi.Tests/RequireHttpsAttributeFixture.cs

```diff
@@ -0,0 +1,144 @@
+﻿namespace Georadix.WebApi
+{
+    using Georadix.WebApi.Filters;
+    using Georadix.WebApi.Testing;
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using System.Web.Http;
+    using Xunit;
+    using Xunit.Extensions;
+
+    public class RequireHttpsAttributeFixture
+    {
+        public const string DeleteResource = BaseUrl + "delete-resource";
+        public const string GetResource = BaseUrl + "get-resource";
+        public const string HeadResource = BaseUrl + "head-resource";
+        public const string PostResource = BaseUrl + "post-resource";
+        public const string PutResource = BaseUrl + "put-resource";
+        private const string BaseUrl = "RequireHttpsAttributeFixtureController/";
+
+        public static IEnumerable<object[]> Scenarios
+        {
+            get
+            {
+                return new object[][]
+                {
+                    new object[] { HttpMethod.Delete, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Get, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Head, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Post, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Put, true, HttpStatusCode.NoContent },
+                    new object[] { HttpMethod.Delete, false, HttpStatusCode.NotFound },
+                    new object[] { HttpMethod.Get, false, HttpStatusCode.Redirect },
```

Comment:
Curious, what's reason behind redirecting 'GET' and 'HEAD' but not 'DELETE' for example?